Traceback (most recent call last):
  File "test.py", line 216, in <module>
    main(args.student_id)
  File "test.py", line 170, in main
    game_results = play_games(student_id, student_ai, f'baseline{i}', baseline_ai)
  File "test.py", line 89, in play_games
    {"name": student_id, "algorithm": student_ai()},
  File "/home/ubitus03/fai_final_project/./b11705043/agent.py", line 298, in setup_ai
    return GBDTAgent()
  File "/home/ubitus03/fai_final_project/./b11705043/agent.py", line 16, in __init__
    self.load_training_data()  # Load training data from file
  File "/home/ubitus03/fai_final_project/./b11705043/agent.py", line 39, in load_training_data
    self.training_data = joblib.load(f)
  File "/home/ubitus03/miniconda3/envs/poker/lib/python3.8/site-packages/joblib/numpy_pickle.py", line 648, in load
    obj = _unpickle(fobj)
  File "/home/ubitus03/miniconda3/envs/poker/lib/python3.8/site-packages/joblib/numpy_pickle.py", line 577, in _unpickle
    obj = unpickler.load()
  File "/home/ubitus03/miniconda3/envs/poker/lib/python3.8/pickle.py", line 1210, in load
    raise EOFError
EOFError
